### PR TITLE
[Spec] Error Message (and Focus) Accessibility

### PR DESCRIPTION
**Pull request by golddove**
## Description
Spec out how errors interact with screen readers.

Current status: Discussion complete, pending implementation.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4929)

### CODE REVIEW

**Review by jwoo-msft — APPROVED** [review]

File: specs/DesignDiscussions/InputAccessibility.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## Error Accessibility
+
+How should a renderer present error messages to screen readers for inputs that have failed validation?
+
+### On Focus
+
+What is announced when focus moves to a validated input?
+
+1. When an Input is in focus, announce the input's label. If the input has failed validation, **append** the error message.
```
Comment (shalinijoshi19) [inline specs/DesignDiscussions/InputAccessibility.md:11]: @golddove to what are we appending the error message corresponding to this first failed validation input ?

Comment (shalinijoshi19) [reply specs/DesignDiscussions/InputAccessibility.md:11]: Can you pull the "decision" option up to the top of the spec as our plan of action so its very clear ? Does the statement above still apply to Option 1 ?

Comment (shalinijoshi19) [reply specs/DesignDiscussions/InputAccessibility.md:11]: Sorry i just realized each option seems to be an option and a decision?

**Comment by paulcam206** [comment]
@golddove - did we close on this spec? if so, we should get it checked in :)

**Comment by jonmill** [comment]
Closing due to age - if this is still needed, please re-open

**Pull request closed without merging**